exercises/11_hashmaps/hashmaps2.rs

```diff
@@ -27,6 +27,14 @@ enum Fruit {
     Pineapple,
 }
 
+const FRUIT_KINDS: [Fruit; 5] = [
```

Comment:
this can be in the test module, and there's no reason to have this be an array when it could be a `Vec<Fruit>` instead